projects/components/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@vcd/ui-components",
-    "version": "13.0.1-dev.1",
+    "version": "13.0.2-dev.1",
```

Comment:
The version number to be bumped up is `dev.2`. In semantic versioning, this is for internal builds. Not that it matters too much since we don't really have too many users and releases. We should still follow some method and [semver](https://semver.org/) is a good guide.